lib/rules/property-allowed-list/README.md

```diff
@@ -4,39 +4,40 @@ Specify a list of allowed properties.
 
 <!-- prettier-ignore -->
 ```css
-a { display: block; }
+a { color: red; }
 /** ↑
  * This property */
 ```
 
-This rule ignores variables (`$sass`, `@less`, `--custom-property`).
+This rule ignores preprocessor variables (`$sass`, `@less`).
+It parses custom properties (`--foo: red`).
 
 The [`message` secondary option](../../../docs/user-guide/configure.md#message) can accept the arguments of this rule.
 
 ## Options
 
 `array|string|regex`: `["array", "of", /properties/, "regex"]|"property"|"/regex/"|/regex/`
 
-If a string is surrounded with `"/"` (e.g. `"/^background/"`), it is interpreted as a regular expression. This allows, for example, easy targeting of shorthands: `/^background/` will match `background`, `background-size`, `background-color`, etc.
+If a string is surrounded with `"/"` (e.g. `"/^background/"`), it is interpreted as a regular expression. This allows, for example, easy targeting of shorthands: `/background/` will match `background`, `background-size`, `background-color`, etc.
```

Comment:
I think that  `/^background/` makes sense because `/background/` would also target prefixed properties.
e.g. `-webkit-background-clip`

″ for property-disallowed-list/README.md

Reply:
Let's revert this change for now. You're welcome to open an issue to discuss it, though. If the consensus is to make the change, we'd want to roll it out consistently across all our rule READMEs.

Reply:
I reverted these changes.